### ISSUE TITLE
increase terminal columns in vmq-admin or add hint about cropping, add more verbosity to vmq-admin session disconnect output

### ISSUE THREAD

**Issue by puresilk** [vernemq/vernemq#1290]
# The Problem
This issue references #1200 

A problem with the default view of ```vmq-admin session show``` is that it crops the client ids, and the mountpoints, and when trying to kick clients (I am doing resilience testing with the MQTT app which is connected, and should reconnect), the command ``` vmq-admin session disconnect ``` will report **Done**, even if no client id by such a name exists. The view gives me the impression that I am executing the correct command, whereas in reality I am trying to kick a non-existing client. 

Here's what I mean:

![image](https://user-images.githubusercontent.com/4606111/62709788-5e7f3880-b9f6-11e9-85a7-db3fddaf1ed9.png)

as you can see the actual client ids are much longer than one might assume. 

I was fumbling around with the short client-ids, wondering why my client was not being disconnected ... 
```vmq-admin session disconnect client-id=js.156527243 -m 5c61a59ba6b76```
Done

then I looked at the client's side and saw that the client id is much longer. 

# Suggestions
## allow the output width to scale to the full terminal width
Possibly difficult as in #1200 @larshesel stated that the cli library does not support this.

## A small hint at the bottom
Add a small hint at the bottom and/or top of the output, something like: "some columns were truncated, please refer to vmq-admin session show --help for information how to access individual columns" 

## Special character to indicate cropping
Add a special character to indicate cropping in relevant fields, e.g. a  … (ellipsis) character or something similar. 
Ideally reference this character in the hint at the bottom ("fields with … were cropped due to limited space")

## Give feedback whether clients were actually disconnected
I believe it would help in debugging, if ``` vmq-admin session disconnect ``` would show the number of clients which were disconnected - if that is difficult to achieve, maybe whether any clients were disconnected as a result of the command.

**Comment by ioolkos** [comment]
@puresilk thanks for your effort & suggestions!

**Comment by joekinley** [comment]
I would also add to this that cropping number of results due to an applied default limit should be indicated in the result. (see https://github.com/vernemq/vernemq/issues/1496)

**Comment by ioolkos** [comment]
@joekinley something like:
`Note that some results might be truncated due to table size` and
`Output Row Limit set to: 100` (or actual row limit that the user has set). Have to check if I can grab that variable there though. But it would appear in every call to `vmq-admin session`

Would that make sense?

**Comment by joekinley** [comment]
Yes that would make sense. I would also appreciate a crop indicator as stated in the ticket though, but at least such message would point the user into the right direction.

**Comment by codeadict** [comment]
I have been experimenting with https://github.com/rabbitmq/stdout_formatter as the output library and so far much better results than clique. It has the ability to wrap the output in the columns and show all the content with a higher column.

**Comment by ioolkos** [comment]
@codeadict looks very interesting. Can it be hooked up to clique somehow? (it's basically an output formatter, not a drop-in replacement for clique which would be near impossible anyway).

**Comment by codeadict** [comment]
I'm trying to see how to hook it up, it's not a clique replacement definitely, it's just an output formatter as you mention, I've played with it a bit for `vmq-admin session show` with a fake data structure, still finding how to integrate it with the output of `vmq-ql`. It's output can be exactly like the `clique` one so using it one command at the time is doable.

**Comment by codeadict** [comment]
Well here is an initial, very untested/raw/wip PR, hoping to make it for next release if we all love it.

**Comment by codeadict** [comment]
@puresilk @joekinley There is a merged PR in the latest release and the `master` branch that addresses the cropping issue. Can you please test and suggest any improvements?

**Comment by puresilk** [comment]
@codeadict thank you very much for your work! 

I have created a v1.10.3 container based on your official Docker image, but patched for v1.10.3, and my own configuration file. I've run into the issue with accept_eula, could not resolve it - will look into it more tomorrow. If there's some suggestions from me on that part (I have one minor point of feedback already) I'll open a new issue for the eula. Otherwise I hope to be able to test the new CLI in the coming days

**Comment by codeadict** [comment]
Awesome @puresilk, also new containers are released for VerneMQ 1.10.3 for easier testing
